src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/notifications/Notifications.java

```diff
@@ -0,0 +1,59 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.notifications;
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import lombok.Getter;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Notifications extends BasePageObject {
+
+  @Getter
+  @FindBy(css = ".wds-notifications__notification-list")
+  private WebElement notificationsList;
+
+  @FindBy(css = ".wds-notification-card")
+  private List<WebElement> notificationCards;
+
+  @FindBy(css = ".wds-notifications__zero-state")
+  private WebElement emptyState;
+
+  @FindBy(css = ".wds-notifications__mark-all-as-read")
+  private WebElement markAllAsRead;
+
+  private final String UNREAD_CLASS = "wds-is-unread";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1170)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this field "UNREAD_CLASS" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00116)